.circleci/config.yml

```diff
@@ -30,6 +30,7 @@ jobs:
         DD_APM_ENABLED: "true"
         DD_BIND_HOST: "0.0.0.0"
         DD_API_KEY: invalid_key_but_this_is_fine
+    - image: memcached:1.5.9
```

Comment:
You might have already noticed, but in case not, we're using `resource_class: xlarge`. I'm not sure how long that'll last us if we keep adding new services. It's 8 CPUs and 16GB.